Apps/WebClient/src/ClientApp/app/views/timeline.vue

```diff
@@ -1,657 +1,638 @@
 <style lang="scss" scoped>
-@import "@/assets/scss/_variables.scss";

```

Comment:
I only made changes on line 96 for new text in banner.  The changes were likely caused by lintfix which I did run as per normal routine.